backend/test/squads.test.js

```diff
@@ -0,0 +1,166 @@
+const chai = require("chai");
+const chaiHttp = require("chai-http");
+chai.use(chaiHttp);
+
+const app = require("../src/index");
+
+// Models
+const User = require('../src/models/user');
+const Squad = require('../src/models/squad');
+
+// Util
+const createTestUser = require('./util/createTestUser');
+const createTestSquad = require('./util/createTestSquad');
+
+const should = chai.should();
+const expect = chai.expect
+
+describe('Squad Tests', () => {
+
+  let token1;
+  let token2;
+
+  let testUser1;
+  let testUser2;
+
+  let testSquad1;
+  let testSquad2;
+
+  let userObject1;
+  let userObject2;
+
+  before(async () => {
+    userObject1 = await createTestUser(
+      'foo1',
+      'bar1',
+      'foo1@bar.com',
+      'foobar1',
+      'foobar1',
+    );
+    userObject2 = await createTestUser(
+      'foo2',
+      'bar2',
+      'foo2@bar.com',
+      'foobar2',
+      'foobar2',
+    );
+
+    testUser1 = userObject1.user;
+    token1 = userObject1.token;
+
+    testUser2 = userObject2.user;
+    token2 = userObject2.token;
+
+    testSquad1 = await createTestSquad(testUser1);
+    testSquad2 = await createTestSquad(testUser1)
```

Comment:
All of this should be removed when I setup the database creation script that will run when booting the application in `test` mode.